lib/process-manager.ts

```diff
@@ -517,6 +525,12 @@ export class QueryProcessManager<T = string, U = string> extends ProcessManager
 		clearTimeout(timeout);
 		return result;
 	}
+	queryTemporaryProcess(input: T, force?: boolean) {
```

Comment:
I don't know how to feel about these function names, mostly.

`queryNewProcess`?